package.json

```diff
@@ -8,9 +8,12 @@
     "compile": "npm run compile:css && npm run compile:js",
     "dev:css": "sass --load-path=node_modules --watch sass/etna.scss:templates/static/css/dist/etna.css",
     "dev:js": "webpack --config webpack.config.js --mode=development --watch",
+    "dev": "npm run dev:css & npm run dev:js",
     "test": "jest --coverage",
-    "lint": "prettier \"{scripts,sass}/**/*.{scss,js}\" --check",
-    "lint:fix": "prettier \"{scripts,sass}/**/*.{scss,js}\" --write"
+    "lint:js": "eslint 'scripts/**/*.js'",
```

Comment:
These have crept in by accident from when I was testing, and I can remove them - or if preferred I could add individual linting for prettier, js and css, and make the lint command call them.

Reply:
I would honestly prefer it all in one command because I can't imagine many scenarios where we'd just want to run one in isolation.

Reply:
I've removed the extra ones.